slamd/design_assistant/processing/llm_service.py

```diff
@@ -257,26 +264,17 @@ def _create_output_format_excerpt(cls):
 
 For you orientation, here are some examples (note that these are just examples and must not be included in the general knowledge provided; further make sure to use units that are conventional for the components you are proposing):
 
-Example 1:
-Water to Cement Ratio: 0.5
+Make sure to stick to exactly this format! Here are examples:
```

Comment:
Bei meinem ersten Test hat er sich nicht an das Format gehalten. Hier das Ergebnis (ich habe relativ viele Komponenten angehakt um auch komplexe Setups zu testen):

Based on the provided design guidelines and components, here is an optimized concrete formulation recipe: - Weight of Powders: 400 kg - Water to Binder Ratio: 35% - Weight of Additives (Biochar, Limestone Powder): 4 kg, 40 kg - Weight of Admixtures (Super Plasticizer): 4 kg - Composition of Powder Blend (Fly Ash/GGBFS): 60/40 - Composition of Liquid (H2O/Activator Liquid): 96/4 - Processing: standard mixing and vibrating, ambient curing - Aggregates: 1800 kg Let's include the components as a comma-separated list: - Weight of Powders: 400 kg - Water to Binder Ratio: 35% - Weight of Additives (Biochar): 4 kg - Weight of Additives (Limestone Powder): 40 kg - Weight of Admixtures (Super Plasticizer): 4 kg - Composition of Powder Blend (Fly Ash/GGBFS): 60/40 - Composition of Liquid (H2O/Activator Liquid): 96/4 - Processing: standard mixing and vibrating, ambient curing - Aggregates: 1800 kg

Reply:
Auch bei einer einfacheren Komposition wurde das Format nicht eingehalten:

- Weight of Powders: 400 kg - Water to Cement Ratio: 40 % - Weight of Recycled Aggregates: 1000 kg - Composition of Powder Blend Fly Ash/GGBFS: 60/40 - Composition of Liquid H2O: 100 % - Processing: standard concrete mixing and curing methods